src/Stripe.net/Entities/_base/StripeEntity.cs

```diff
@@ -27,29 +25,7 @@ public abstract class StripeEntity : IStripeEntity
         /// </remarks>
         /// <returns>The raw <see cref="JObject">JObject</see>.</returns>
         [JsonIgnore]
-        public JObject RawJObject
-        {
-            get
-            {
-                // Lazily initialize the object the first time the getter is called.
-                if (this.rawJObject == null)
-                {
-                    if (this.StripeResponse == null)
-                    {
-                        return null;
-                    }
-
-                    this.rawJObject = JObject.Parse(this.StripeResponse.Content);
-                }
-
-                return this.rawJObject;
-            }
-
-            protected set
-            {
-                this.rawJObject = value;
-            }
-        }
+        public JObject RawJObject { get; set; }
```

Comment:
I'm sorry I missed this in the original review. Can we keep the setter accessibility the same as before by defining an internal method that `StripeEntityConverter` will call to set the value?